api/auth_test.go

```diff
@@ -119,32 +96,18 @@ func TestAuthNotAuthorizedWNoACLs(t *testing.T) {
 	etreurl = server.url + etre.API_ROOT + "/entity/" + entityType
 	newEntity := etre.Entity{"host": "local"}
 	payload, err := json.Marshal(newEntity)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 	var gotWR etre.WriteResult
 	statusCode, err = test.MakeHTTPRequest("POST", etreurl, payload, &gotWR)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if statusCode != http.StatusForbidden {
-		t.Errorf("response status = %d, expected %d", statusCode, http.StatusForbidden)
-	}
-	if gotWR.Error == nil {
-		t.Errorf("WriteResult.Error is nil, expected error message")
-	} else if gotWR.Error.Type != "not-authorized" {
-		t.Errorf("WriteResult.Error.Type = %s, expected not-authorized", gotWR.Error.Type)
-	}
-
-	if diff := deep.Equal(gotCaller, caller); diff != nil {
-		t.Error(diff)
-	}
+	require.NoError(t, err)
+	assert.Equal(t, http.StatusForbidden, statusCode)
+	require.NotNil(t, gotWR.Error)
+	assert.Equal(t, "not-authorized", gotWR.Error.Type)
```

Comment:
Probably out of scope of this PR, but the inlined "not-authorized" probably should be a package level constant somewhere. As does with the error type strings in tests throughout this PR.